addons/website_forum/views/website_forum.xml

```diff
@@ -735,7 +735,12 @@
         </div>
         <div class="question-block">
             <div class="pull-right author-box">
-                <img class="pull-left img img-rounded img-avatar" t-attf-src="/forum/user/#{answer.create_uid.id}/avatar"/>
+                <div class="user-tooltip" t-if=" answer.create_uid.karma >= answer.forum_id.karma_expand_userbio">
+                    <t t-call="website_forum.forum_user_tooltip"/>
+                </div>
+                <a t-attf-href="/forum/#{ slug(forum) }/user/#{ answer.create_uid.id }" class="user-info">
```

Comment:
Please prefix and use correct naming in css classes, like o_forum_user_info. Update all classes you added if necessary.